add_kdrama.php

```diff
@@ -0,0 +1,26 @@
+<?php
+require_once "functions.php";
+
+var_dump($_POST);
+
+if (
+    isset($_POST['Kdrama']) &&
+    isset($_POST['ReleaseYear']) &&
+    isset($_POST['Synopsis']) &&
+    isset($_POST['Starring'])
+) {
+    $db = getDb();
+
+    $Kdrama = $_POST['Kdrama'];
+    $Releaseyear = $_POST['ReleaseYear'];
+    $Synopsis = $_POST['Synopsis'];
+    $Starring = $_POST['Starring'];
+
+    $query = $db->prepare('INSERT INTO `Kdramas` (`Kdrama`, `ReleaseYear`, `Synopsis`, `Starring`) VALUES (:Kdrama, :ReleaseYear, :Synopsis, :Starring);');
+
+    $query->bindParam('Kdrama', $Kdrama);
+    $query->bindParam('ReleaseYear', $Releaseyear);
+    $query->bindParam('Synopsis', $Synopsis);
+    $query->bindParam('Starring', $Starring);
+    $query->execute();
```

Comment:
This all works perfectly!

It might be nice to send the user back to the home page after the drama has been saved, you could do this by adding the following code after the query gets executed
```
header('Location: index.php');
exit;
```

This will re-direct the user back home so they can see the new drama they added.